src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedEditorBehaviorFeature.java

```diff
@@ -55,4 +55,11 @@ public boolean isEnableEnterBetweenBracesFix(@NotNull PsiFile file) {
         // Note that this defaults to enabled for user-defined language server definitions
         return (editorSettings == null) || editorSettings.enableEnterBetweenBracesFix;
     }
+
+    @Override
+    public boolean isEnableTextMateNestedBracesImprovements(@NotNull PsiFile file) {
```

Comment:
And the overridden logic for user-defined language server definitions that consults client config and defaults to enabled.